src/gc/objecthandle.cpp

```diff
@@ -21,12 +21,14 @@
 
 #include "gchandletableimpl.h"
 
-#ifdef FEATURE_COMINTEROP
-#include "comcallablewrapper.h"
-#endif // FEATURE_COMINTEROP
-#ifndef FEATURE_REDHAWK
-#include "nativeoverlapped.h"
-#endif // FEATURE_REDHAWK
+#ifndef BUILD_AS_STANDALONE
```

Comment:
A nit - could you please revert the indentation of the `#include`s and `#ifdef`s? We don't do that in most of the code.